### PR TITLE
Adding deselect-all and balatro-safety mods.

### PR DESCRIPTION
**Pull request by Zei33**

### CODE REVIEW

File: index.json [hunk 1]
```diff
@@ -10,5 +10,31 @@
     "load_before": [],
     "load_after": [],
     "authors": ["Dogeek <simon.bordeyne@gmail.com>"]
+  },
+  {
+    "url": "deselect-all.tar.gz",
```
Comment (sbordeyne) [inline index.json]: that's not an URL to the archive

File: index.json [hunk 2]
```diff
@@ -10,5 +10,31 @@
...
+    "description": [
+      "Adds a button to deselect all highlighted cards."
+    ],
+    "version": "1.0.0",
+    "load_before": [],
+    "load_after": [],
+    "authors": ["Zei33 <balatro@zei.gg>"]
+  },
+  {
+    "url": "balatro-safety.tar.gz",
```
Comment (sbordeyne) [inline index.json]: same, not an url

File: index.json [hunk 3]
```diff
@@ -10,5 +10,31 @@
...
+    "id": "balatro-safety",
+    "name": "Balatro Safety",
+    "description": [
+		"Protects you from accidentally making",
```
Comment (sbordeyne) [inline index.json]: avoid having so much whitespace

**Review by sbordeyne — CHANGES_REQUESTED** [review]
address the issues, as is the mods won't be downloadable from the menu

**Comment by Zei33** [comment]
@sbordeyne thanks I was unclear on the process. I will resolve the issues tonight.

File: index.json (hunk 3 above)
Comment (Zei33) [reply index.json]: This is based on tab length. I will change my editor to use spaces instead of tabs.

**Comment by Zei33** [comment]
> address the issues, as is the mods won't be downloadable from the menu

@sbordeyne the changes have been made. It should be right now.

**Review by sbordeyne — APPROVED** [review]

**Pull request merged**